setup.cfg

```diff
@@ -2,4 +2,16 @@
 universal=1
 
 [flake8]
+exclude=*migrations*
 ignore=E122,E123,E124,E125,E126,E127,E128,E129,E131,E201,E202,E203,E221,E225,E226,E231,E241,E251,E261,E262,E265,E271,E301,E302,E303,E501,E502,E701,E713,E714,F401,F403
+max-line-length = 99
```

Comment:
i thought we agreed on 99, but checking the documentation is still 79. @mkoistinen @FinalAngel @evildmp which size do we want? Any is fine for me (the longer the better from my POV)